include/cripts/Lulu.hpp

```diff
@@ -74,7 +74,6 @@ template <typename ChildT> class StringViewMixin
 
```

Comment:
Seems like this should be:
```
constexpr StringViewMixin(const Cript::string_view &str) { _value = str; }
```
or just pass by value.

Reply:
Probably. Alan kinda convinced me that these gets optimized in funny ways, and I also couldn't get a good answer if it should be passed by reference or not.

Reply:
The short rule of thumb (since C++17) is, `f(T v)` and `f(const T &v)` are generally the same if the function is inlined and doesn't change `v`.  With `T g();`, `f(g())` works for both definitions of `f`, because C++ allows a const reference to refer to an unnamed temporary.  But C++ doesn't allow a non-const reference to refer to an unnamed temporary.